packages/package_bumbershoot_3_0_8738/.shed.yml

```diff
@@ -1,11 +0,0 @@
-categories:
-- Tool Dependency Packages
-description: Contains a tool dependency definition that downloads the Bumbershoot tools.
-long_description: |
```

Comment:
@chambm can you leave the packages please. Maybe we need to fix some older version. It is perfectly fine to remove the tool_dependencyies.xml file and completely go with conda-only, but I would leave the package for the moment.